Examples/Io/Csv/src/CsvMultiTrajectoryWriter.cpp

```diff
@@ -0,0 +1,207 @@
+
+  // Counter of truth-matched reco tracks
+  using RecoTrackInfo = std::pair<trackInfo, size_t>;
+  std::map<ActsFatras::Barcode, std::vector<RecoTrackInfo>> matched;
+
+  size_t track_id = 0;
+  for (const auto& traj : trajectories) {
+    // The trajectory entry indices and the multiTrajectory
+    const auto& trackTips = traj.tips();
+    const auto& mj = traj.multiTrajectory();
+    if (trackTips.empty()) {
+      ACTS_WARNING("Empty multiTrajectory.");
+      continue;
+    }
+
+    // Loop over all trajectories in a multiTrajectory
+    for (const size_t& trackTip : trackTips) {
+      // Collect the trajectory summary info
+      auto trajState =
+          Acts::MultiTrajectoryHelpers::trajectoryState(mj, trackTip);
+      // Reco track selection
+      //@TODO: add interface for applying others cuts on reco tracks:
+      // -> pT, d0, z0, detector-specific hits/holes number cut
+      if (trajState.nMeasurements < m_cfg.nMeasurementsMin) {
+        continue;
+      }
+
+      // Check if the reco track has fitted track parameters
+      if (not traj.hasTrackParameters(trackTip)) {
+        ACTS_WARNING(
+            "No fitted track parameters for trajectory with entry index = "
+            << trackTip);
+        continue;
+      }
+
+      // Get the majority truth particle to this track
+      std::vector<ParticleHitCount> particleHitCount;
+      identifyContributingParticles(hitParticlesMap, traj, trackTip,
+                                    particleHitCount);
+      if (particleHitCount.empty()) {
+        ACTS_WARNING(
+            "No truth particle associated with this trajectory with entry "
+            "index = "
+            << trackTip);
+        continue;
+      }
+
+      // Requirement on the pT of the track
+      const auto& momentum = traj.trackParameters(trackTip).momentum();
+      const auto pT = Acts::VectorHelpers::perp(momentum);
+      if (pT < m_cfg.ptMin) {
+        continue;
+      }
+
+      // Get the majority particle counts
+      ActsFatras::Barcode majorityParticleId =
+          particleHitCount.front().particleId;
+      // n Majority hits
+      size_t nMajorityHits = particleHitCount.front().hitCount;
+
+      // track info
+      trackInfo toAdd;
+      toAdd.track_id = track_id;
+      toAdd.particleId = majorityParticleId;
+      toAdd.nMajorityHits = nMajorityHits;
+      toAdd.nMeasurements = trajState.nMeasurements;
+      toAdd.nOutliers = trajState.nOutliers;
+      toAdd.nHoles = trajState.nHoles;
+      toAdd.chi2Sum = trajState.chi2Sum * 1.;
+      toAdd.NDF = trajState.NDF;
+      toAdd.truthMatchProb = toAdd.nMajorityHits * 1. / trajState.nMeasurements;
+      toAdd.fitterParameters = &traj.trackParameters(trackTip);
+      toAdd.trackType = "unknown";
+
+      // Check if the trajectory is matched with truth.
+      if (toAdd.truthMatchProb >= m_cfg.truthMatchProbMin) {
+        matched[toAdd.particleId].push_back({toAdd, toAdd.track_id});
+      } else {  // FIX ME
```

Comment:
What does the `FIX ME` mean?

Reply:
Nothing. I forgot to remove it when I was writing the code.